micropub.php

```diff
@@ -383,32 +404,39 @@ private static function update() {
 				}
 				$delete_args = static::mp_to_wp( array( 'properties' => $delete ) );
 				if ( $delete_args['tags_input'] ) {
-					$args['tags_input'] = array_diff( $args['tags_input'] ?: array(),
-													  $delete_args['tags_input']);
+					$args['tags_input'] = array_diff(
+						$args['tags_input'] ?: array(),
+						$delete_args['tags_input']
+					);
 				}
 				if ( $delete_args['post_category'] ) {
-					$args['post_category'] = array_diff($args['post_category'] ?: array(),
-														 $delete_args['post_category']);
+					$args['post_category'] = array_diff(
+						$args['post_category'] ?: array(),
+						$delete_args['post_category']
+					);
 				}
 			} elseif ( is_array( $delete ) ) {
 				foreach ( static::mp_to_wp( array( 'properties' => array_flip( $delete ) ) )
-						  as $name => $_ ) {
-					$args[ $name ] = NULL;
+					as $name => $_ ) {
+					$args[ $name ] = null;
 				}
-				if ( in_array( 'category', $delete ) ) {
-					wp_set_post_tags( $post_id, '', false);
-					wp_set_post_categories( $post_id, '');
+				if ( in_array( 'category', $delete, true ) ) {
+					wp_set_post_tags( $post_id, '', false );
+					wp_set_post_categories( $post_id, '' );
 				}
 			} else {
 				static::error( 400, 'delete must be an array or object' );
 			}
 		}
 
-		// tell wordpress to preserve published date explicitly, otherwise
+		// tell WordPress to preserve published date explicitly, otherwise
 		// wp_update_post sets it to the current time
 		$args['edit_date'] = true;
-		$args = static::store_mf2( static::store_geodata(
-			static::generate_post_content( $args ) ) );
+		$args              = static::store_mf2(
```

Comment:
so ugly :(

Reply:
I hated this line for different reasons beforehand. This is where I wanted to change it back to having these functions act as filters, but I think the three layers of functions in 1 entry is confusing in the first place. For now, I think I will convert them to 3 separate lines for readability.

Reply:
ok!